assets/js/pages/Form.tsx

```diff
@@ -12,74 +12,44 @@ import {
   Link,
 } from "@chakra-ui/react";
 import React from "react";
+import { checkEventForm, sendEventForm } from "../utils/formApi";
 
-type EventInfo = {
+export type EventForm = {
   title: string;
   description: string;
   start_date: string;
   presenters: string[];
 };
 
-type Links = {
+export type Links = {
   viewer_link: string;
   moderator_link: string;
 };
 
 function Form() {
```

Comment:
Based on how many input fields and its states we have to handle here I would recommend to start using a true form :D And the easiest way to i implement and handle forms in react is to use dedicated library (e.g Formik or React-hook-form are great tools)
What I found  - chackra UI integrates with Formik in very nice way. Here example of use https://chakra-ui.com/getting-started/with-formik so to be considered ;)